tests/dynamic_data/users_roles/delete_user_test.py

```diff
@@ -1,47 +1,104 @@
 import pytest
 from layman import app, util as layman_util
-from test_tools import process_client
+from layman.http import LaymanError
+from layman_settings import ANONYM_USER, NONAME_USER
+from test_tools import process_client, role_service, process
 
 
-USERNAME = 'test_delete_user'
+@pytest.fixture(scope='module')
+def _setup_env_vars():
+    env_vars = dict(process.AUTHN_SETTINGS)
+    env_vars['GRANT_DELETE_OTHER_USER'] = "test_delete_user2,ADMIN"
+    process.ensure_layman_function(env_vars)
```

Comment:
We are currently not clearing role `ADMIN` after the test and I would like to clear it. It can be done e.g. in this fixture:
```
yield
role_service.delete_role('ADMIN')
```